util/regtool.py

```diff
@@ -70,9 +68,10 @@ def main():
     parser.add_argument('-f',
                         action='store_true',
                         help='Output as FPV CSR rw assertion module')
-    parser.add_argument('--outdir', '-t',
-                        help='Target directory for generated RTL, '\
-                             'tool uses ../rtl if blank.')
+    parser.add_argument('--outdir',
+                        '-t',
+                        help='Target directory for generated RTL, '
```

Comment:
nit: Not a Python issue, I realise, but I think this should be a semicolon not a comma.